### PR TITLE
Super shoot

### PR DESCRIPTION
**Pull request by wheesko**

### CODE REVIEW

**Review by KernalPanik — APPROVED** [review]
Needs some tweaks, but looks good

File: Assets/Scripts/Player/PlayerController.cs [hunk 1]
```diff
@@ -146,7 +146,7 @@ private void Update()
 
             this.transform.rotation = Quaternion.Euler(0, rotY, 0);
             playerView.rotation = Quaternion.Euler(rotX, rotY, 0);
-            weaponGameObject.transform.rotation = Quaternion.Euler(rotX, rotY, 0);
+            //weaponGameObject.transform.rotation = Quaternion.Euler(rotX, rotY, 0);
```
Comment (salmonela7) [inline Assets/Scripts/Player/PlayerController.cs]: Remove this comment if not needed

File: Assets/Scripts/Weapons/SuperShotgun.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    public AudioClip hitEnemyClip;
+    public GameObject Bullet;
+    public int Damage;
+    public float fireRate;
+    private float nextFire;
+    private AudioSource audioSource;
+
+    void Start()
+    {
+        Debug.Log(transform.position);
```
Comment (salmonela7) [inline Assets/Scripts/Weapons/SuperShotgun.cs]: Delete debug.log :D

**Review by salmonela7 — APPROVED** [review]
Good work! 💯

**Pull request merged**